OpenRVS/classes/OpenBeacon.uc

```diff
@@ -4,24 +4,46 @@
 //IMPORTANT: Servers with N4 Admin should not install this class
 //debug - should log any time an initial connection with client opens
 //installed server side
-
 class OpenBeacon extends UdpBeacon transient;
 
+// REGISTRY refers to a server running github.com/ijemafe/openrvs-registry.
+// This can be updated here, or disabled by commenting the RegisterServer() call
+// in OpenServer.uc.
+const REGISTRY_IP = "64.225.54.237";//Current openrvs-registry deployment

```

Comment:
Can we make these config variables, with this as the default?  Keeps it consistent with fetching the server list, and also means that a recompile is not needed to change your registry provider.